hypha/apply/funds/models/applications.py

```diff
@@ -232,6 +233,18 @@ class RoundBase(WorkflowStreamForm, SubmittableStreamForm):  # type: ignore
     )
     sealed = models.BooleanField(default=False)
 
+    def get_url(self) -> Optional[str]:
+        """Generates the live url, primarily used in the wagtail admin for the "view live" button.
+
```

Comment:
The only issue I had was when I went from my dashboard (`/dashboard/`) to the Wagtail admin dashboard (`/admin`) I got the error `RoundBase.get_url() got an unexpected keyword argument 'request'`. This seems to fix it just by adding the optional request kwarg. All looks good otherwise!

```suggestion
    def get_url(self, request: Optional[WSGIRequest]=None) -> Optional[str]:
        """Generates the live url, primarily used in the wagtail admin for the "view live" button.

        Args:
            request: The request that will be passed by Wagtail when loading the admin interface, currently unused.
```

Reply:
Nice catch, I thought I could get away without passing all the original parameters :D

Adding them.

Reply:
Done, https://github.com/HyphaApp/hypha/pull/3805/commits/b416c2d3a61356793b32df82509d041b7725d36b